apps/src/templates/manageStudents/ManageStudentsActionsCell.jsx

```diff
@@ -21,10 +21,12 @@ class ManageStudentActionsCell extends Component {
     id: PropTypes.number.isRequired,
     sectionId: PropTypes.number.isRequired,
     isEditing: PropTypes.bool,
+    isSaving: PropTypes.bool,
```

Comment:
You haven't hooked up the `isSaving` prop in your redux connect method.

Reply:
I'm keeping this a per student flag, because there will both be the ability to save an individual student, and a group of them in the table.